src/domain/app/App.tsx

```diff
@@ -34,8 +36,13 @@ class App extends React.Component<AppProps> {
           this.props.fetchApiTokenError({ message: 'No user found' });
         }
       })
-      .catch(error => this.props.fetchApiTokenError(error));
+      .catch(error => {
+        this.props.fetchApiTokenError(error);
+        console.log('App.tsx loaduser catch error');
+        console.error(error);
```

Comment:
toast it?

Reply:
I think we can't toast this, it shows up randomly for returning users. Removing the error logs for now.